feature/gribi/ate_tests/route_removal_non_default_vrf_test/route_removal_non_default_vrf_test.go

```diff
@@ -97,12 +97,27 @@ func TestRouteRemovalNonDefaultVRFFlush(t *testing.T) {
 	ctx := context.Background()
 
 	dut := ondatra.DUT(t, "dut")
+
+	// For interface configuration, Arista prefers config Vrf first then the IP address
+	switch dut.Vendor() {
+	case ondatra.ARISTA:
+		configureNetworkInstance(t, dut)
```

Comment:
Please can we explain why this is order dependent only for a single vendor? If it's to do with references being validated then shouldn't it be generic? If it's not, then this should likely be behind a deviation (I generally don't expect that we have `switch dut.Vendor()` without a deviation unless it's for a naming specific thing).